app/src/components/common/layout/header/index.tsx

```diff
@@ -174,6 +191,8 @@ const HeaderContainer: React.FC = (props: any) => {
   const [isYourConnectionModalOpen, setYourConnectionModalState] = useState(false)
   const [isDepositWithdrawModalOpen, setDepositWithdrawModalState] = useState(false)
   const [depositWithdrawType, setDepositWithdrawType] = useState<ExchangeType>(ExchangeType.deposit)
+  const [governancePage, setGovernancePage] = useState(false)
+  const [marketPage, setMarketPage] = useState(true)
```

Comment:
@gem1no Hmm I think you need just one useState for both of them here...

Reply:
hah yeah oops thanks!